testemci.json

```diff
@@ -3,31 +3,31 @@
   "test_page": "tests/index.html",
   "launchers": {
     "SauceLabs_Chrome": {
-      "command": "saucie --platformSL='Windows 7' --host='http://localhost' --port=8080",
+      "command": "./node_modules/.bin/ember-cli-sauce --platformSL='Windows 7' --no-ct -u <url>",
```

Comment:
This token gets replaced by testem with "http://localhost:8080/someId", so you don't need to configure the host & port manually.